packages/openapi-ts/src/utils/handlebars.ts

```diff
@@ -128,17 +128,24 @@ const dataDestructure = (config: Config, operation: Operation) => {
     return '';
 };
 
+export const getDefaultPrintable = (p: OperationParameter | Model): string | undefined => {
```

Comment:
Added helper here for now, as the default is no longer always a string but `unknown` instead. this is more proper